app/helpers/application_formatters_helper.rb

```diff
@@ -11,15 +11,15 @@ module ApplicationFormattersHelper
   # @param [String] text The text to display.
   # @return [String]
   def format_block_text(text)
-    format_html(simple_format(text, {}, sanitize: false))
+    simple_format(html_escape(text), {}, sanitize: false)
   end
 
   # Formats the given user-input string. The string is assumed not to contain HTML markup.
   #
   # @param [String] text The text to display.
   # @return [String]
   def format_inline_text(text)
-    format_html(html_escape(text))
+    html_escape(text)
```

Comment:
@jeremyyap Can you compare the results of `html_escape("<script>xxx</script>")` and `format_html(html_escape("<script>xxx</script>"))` ?  I suspect the later one will display the script tags but not executing it.